app/src/main/java/com/codesoom/assignment/models/HttpMethod.java

```diff
@@ -0,0 +1,15 @@
+package com.codesoom.assignment.models;
+
+public enum HttpMethod {
```

Comment:
이런 `enum`에는 RFC 문서를 링크하면 좋겠죠!
```suggestion
/**
 * HTTP 프로토콜의 메소드를 정의합니다.
 * @link https://datatracker.ietf.org/doc/html/rfc2616#section-9
 */
public enum HttpMethod {
```

Reply:
넵 StatusCode enum에도 RFC문서 첨부하였습니다!